### PR TITLE
📚 Update changelog

### PR DESCRIPTION
**Pull request by CLiu13**
Before raising the PR, here is a check list:

1) have you written unit tests for your code changes?
2) have you run "make format"?
3) are you requesting to "dev"?
4) have you updated the change log?
5) do you think that you can understand your changes after 6 month?
   5.1) can someone else understand your changes without your explanation?
6) are you pround of your code changes?
   6.1) do you have the feeling of achievement?

### CODE REVIEW

**Comment by CLiu13** [comment]
I read through the changelog and cannot find any entry describing the addition of handlebars as an engine option - would you also like me to add that to the changelog?

**Comment by PrajwalM2212** [comment]
@CLiu13  We do not intend to include handlebars in the main repo. Infact we will be moving it to a separate repo.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Change log
 ================================================================================
 
+0.3.4 - 14-11-2018
+--------------------------------------------------------------------------------
+
+Added
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. global variables to store the target and template files in the jinja2 engine
```
Comment (PrajwalM2212) [inline CHANGELOG.rst]: `target and template files` to `target and template file names`

**Comment by chfw** [comment]
@CLiu13 , the change log looks OK. please do this:

1) update it in .moban.cd/changelog.yml
2) then call `make update`, which auto generates changelog.rst

Otherwise, next `make update` will overwrite your change log. 

I know it is **new** piece of information. Please feel free to update docs.

**Comment by CLiu13** [comment]
Thanks for the info on the correct way to update the changelog. Running `make update` resulted in some additional changes in other files, which I have included for now - but let me know if you would like me to exclude them.

**Pull request merged**